tests/download_tools.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Seems that you made a `_utils` directory. Then how about moving it to this directory?